docs/self-hosted-appcircle/configure-server/advanced-configuration/store-dist-dmz.md

```diff
@@ -0,0 +1,605 @@
+```
+dist.appcircle.spacetech.com
+```
+
+  </TabItem>
+
+</Tabs>
+
+- Check the authentication domain of the Appcircle server.
+
+```bash
+yq '.keycloak.external.domain' ./projects/spacetech/export/.global.yaml
+```
+
+Output:
+
+```
+auth.appcircle.spacetech.com
+```
+
+- Check the API domain of the Appcircle server.
+
+```bash
+yq '.apiGateway.external.domain' ./projects/spacetech/export/.global.yaml
+```
+
+Output:
+
+```
+api.appcircle.spacetech.com
+```
+
+According to the sample outputs above, the needed domains that clients accessing via the internet should know are as follows:
+
+- `store.spacetech.com`: Custom Enterprise App Store domain.
+- `dist.spacetech.com`: Custom Testing Distribution domain.
+- `auth.appcircle.spacetech.com`: Appcircle authentication domain.
+
+Also the Appcircle DMZ server should be resolving some of the Appcircle server domains such as authentication and API domains.
+
+These domains should be resolved to the Appcircle server IP. The domains may vary according to the Appcircle server configuration.
+
+According to the sample outputs above, the needed domains that Appcircle DMZ server should know are as follows:
+
+- `api.appcircle.spacetech.com`: Appcircle API domain.
+- `auth.appcircle.spacetech.com`: Appcircle authentication domain.
+
+:::caution
+There is a common domain here. Be aware that the auth domain that the clients that will access from the internet and the Appcircle DMZ server will connect to should mean two different things.
+:::
+
+## Creating the Appcircle DMZ Server Configuration
+
+To create the Appcircle DMZ server configuration, you should login to the Appcircle server.
+
+You will create all the configuration files on the Appcircle server and then move the created configuration files to the Appcircle DMZ server.
+
+To create the Appcircle DMZ server configuration, you can follow the steps below.
+
+- Login to the Appcircle server with SSH.
+
+- Go to the Appcircle server directory.
+
+```bash
+cd appcircle-server
+```
+
+- Stop the running Appcircle server.
+
+<SpacetechExampleInfo />
+
+```bash
+./ac-self-hosted.sh -n spacetech down
+```
+
+- Create the new configuration files for the Appcircle DMZ and the Appcircle server.
+
+```bash
+./ac-self-hosted.sh -n spacetech export --dmz
+```
+
+:::info
+The `--dmz` flag in the `export` subcommand above creates the configuration files according to your `global.yaml` file of your project for the both the Appcircle DMZ server and the Appcircle server.
+:::
+
+- Check the exported DMZ directory that contains the required files to create Appcircle DMZ server.
+
+```bash
+ls -lah projects/spacetech/export/dmz/
+```
+
+- Start the Appcircle server.
+
+```bash
+./ac-self-hosted.sh -n spacetech up
+```
+
+- Compress the directory into a tarball in the Appcircle server.
+
+```bash
+tar -czf dmz.tar.gz -C projects/spacetech/export/dmz/ .
+```
+
+- Transfer the `dmz.tar.gz` file to the Appcircle DMZ server with a file transfer protocol like `scp` or `ftp`.
```

Comment:
This document should have a an **important caution box** in the proper location that explains that **not only Appcircle server upgrades but also configuration changes** (`global.yaml`) on the private Appcircle server might require re-export of the DMZ server.

So, **on every configuration change**, the whole architecture should be considered like single-node solution and DMZ export should also be done.

For example, when you make the below configuration changes for Appcircle server it the private network, DMZ server should also get configuration updates for proper working.

- Monitoring `enabled` ➡️ `disabled`
- Monitoring `disabled` ➡️ `enabled`